diff --git a/a b/a
new file mode 100644
index 0000000..4ef30bb
--- /dev/null
+++ b/a
@@ -0,0 +1 @@
+file a
diff --git a/b b/b
new file mode 100644
index 0000000..4f2e652
--- /dev/null
+++ b/b
@@ -0,0 +1 @@
+file b
diff --git a/c.sh b/c.sh
new file mode 100644
index 0000000..b35efa3
--- /dev/null
+++ b/c.sh
@@ -0,0 +1 @@
+echo c
diff --git a/d.sh b/d.sh
new file mode 100755
index 0000000..85a69f5
--- /dev/null
+++ b/d.sh
@@ -0,0 +1 @@
+echo d
diff --git a/e.sh b/e.sh
new file mode 100755
index 0000000..209da8d
--- /dev/null
+++ b/e.sh
@@ -0,0 +1,2 @@
+#!/bin/sh
+echo e
diff --git a/f.py b/f.py
new file mode 100644
index 0000000..4606edb
--- /dev/null
+++ b/f.py
@@ -0,0 +1 @@
+print 1
diff --git a/g.py b/g.py
new file mode 100755
index 0000000..c5b1e55
--- /dev/null
+++ b/g.py
@@ -0,0 +1,2 @@
+#!/usr/bin/python
+print 1
diff --git a/h.py b/h.py
new file mode 100644
index 0000000..c5b1e55
--- /dev/null
+++ b/h.py
@@ -0,0 +1,2 @@
+#!/usr/bin/python
+print 1
